python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "pylance"
-dependencies = ["pyarrow>=10", "numpy>=1.22"]
+dependencies = ["pyarrow>=13", "numpy>=1.22"]
```

Comment:
for `pa.FixedShapeTensorType`

Reply:
hmm maybe we shouldn't require this for now. We might actually want to wait a few versions, since users often have other PyArrow requirements that will cause conflicts.

How hard would it be refactor so we only import `FixedShapeTensorType` if PyArrow `>= 13`?

Reply:
probably doable in 2 hrs.

Reply:
can we switch this back?